saas/settings.py

```diff
@@ -98,6 +98,7 @@
         'static', 'data', 'currencies.json'),
     'DEFAULT_UNIT': 'usd',
     'DISABLE_UPDATES': False,
+    'DISPLAY_GROUP_BUY': True,
```

Comment:
Maybe call the settings 'DISPLAY_BULK_BUYER_TOGGLE'. "Group Buy" is what we have called the feature in documentation but the code always refer to it as 'bulk_buyer'.